RIPS/rip-7712.md

```diff
@@ -90,47 +90,23 @@ Before the first RIP-7560 transaction type defined validation frame is applied d
 the `NonceManager` is invoked with the following data:
 
 ```
-sender{20 bytes} nonceKey{24 bytes} nonceSeq{8 bytes}
+sender{20 bytes} nonceKey{32 bytes} nonceSequence{32 bytes}
 ```
 
 The gas costs of this validation frame are counted towards the total for a transaction,
 and is deducted from the account's `validationGasLimit`.
 
-#### NonceManager Pseudocode
+#### Nonce query frame
 
-```
```

Comment:
scorched earth: don't remove old pseudo code before putting in a new solidity code.